extensions/typescript-language-features/src/features/fileConfigurationManager.ts

```diff
@@ -186,7 +186,7 @@ export default class FileConfigurationManager extends Disposable {
 			importModuleSpecifierPreference: getImportModuleSpecifierPreference(config),
 			importModuleSpecifierEnding: getImportModuleSpecifierEndingPreference(config),
 			allowTextChangesInNewFiles: document.uri.scheme === fileSchemes.file,
-			providePrefixAndSuffixTextForRename: config.get<boolean>('renameShorthandProperties', true),
+			providePrefixAndSuffixTextForRename: config.get<boolean>('useAliasesForRenames', true),
```

Comment:
This should read both the old and new setting value for migration

Reply:
Thanks you guidance! This was an attempt at my first PR. I'll keep this in mind with my future contributions.

Reply:
Until we remove the deprecated setting, we should make sure we respect previous setting value. Something like:

```ts
config.get<boolean>('renameShorthandProperties', true) === false ? false : config.get<boolean>('useAliasesForRenames', true)
```

Reply:
I apologize. Not sure how I missed this in the previous attempt at fixing my first commit.